src/azul/queues.py

```diff
@@ -434,13 +434,17 @@ def submit(f, *args, **kwargs):
                 if queue_name == config.notifications_queue_name():
                     submit(self._manage_lambda, config.indexer_name, enable)
                     submit(self._manage_sqs_push, config.indexer_name + '-contribute', queue, enable)
+                elif queue_name == config.notifications_queue_name(retry=True):
+                    submit(self._manage_sqs_push, config.indexer_name + '-contribute_retry', queue, enable)
                 elif queue_name == config.tallies_queue_name():
                     submit(self._manage_sqs_push, config.indexer_name + '-aggregate', queue, enable)
                 elif queue_name == config.tallies_queue_name(retry=True):
                     # FIXME: Brittle coupling between the string literal below and
                     #        the handler function name in app.py
                     #        https://github.com/DataBiosphere/azul/issues/1848
                     submit(self._manage_sqs_push, config.indexer_name + '-aggregate_retry', queue, enable)
+                else:
+                    assert False
```

Comment:
```suggestion
                    assert False, queue_name
```
However, the assertion can't be used as the `if` block doesn't cover all queue names:
```
(azul2) daniel@Crispin ~/azul2 $ python scripts/manage_queues.py purge_all
2021-03-23 10:18:30,818 INFO    MainThread: Found credentials in environment variables.
2021-03-23 10:18:31,785 INFO    ThreadPoolExecutor-0_2: Found credentials in environment variables.
2021-03-23 10:18:31,788 INFO    ThreadPoolExecutor-0_0: Found credentials in environment variables.
2021-03-23 10:18:31,789 INFO    ThreadPoolExecutor-0_3: Found credentials in environment variables.
2021-03-23 10:18:31,790 INFO    ThreadPoolExecutor-0_1: Found credentials in environment variables.
2021-03-23 10:18:31,791 INFO    ThreadPoolExecutor-0_4: Found credentials in environment variables.
2021-03-23 10:18:32,399 WARNING ThreadPoolExecutor-0_0: azul-indexer-daniel is already disabled.
2021-03-23 10:18:32,428 INFO    ThreadPoolExecutor-0_3: Push from 'https://queue.amazonaws.com/122796619775/azul-tallies-daniel.fifo' to lambda function 'azul-indexer-daniel-aggregate' is in state 'Disabled'.
2021-03-23 10:18:32,705 INFO    ThreadPoolExecutor-0_2: Push from 'https://queue.amazonaws.com/122796619775/azul-notifications_retry-daniel' to lambda function 'azul-indexer-daniel-contribute_retry' is in state 'Disabled'.
2021-03-23 10:18:32,712 INFO    ThreadPoolExecutor-0_1: Push from 'https://queue.amazonaws.com/122796619775/azul-notifications-daniel' to lambda function 'azul-indexer-daniel-contribute' is in state 'Disabled'.
2021-03-23 10:18:32,743 INFO    ThreadPoolExecutor-0_4: Push from 'https://queue.amazonaws.com/122796619775/azul-tallies_retry-daniel.fifo' to lambda function 'azul-indexer-daniel-aggregate_retry' is in state 'Disabled'.
Traceback (most recent call last):
  File "scripts/manage_queues.py", line 93, in <module>
    main(sys.argv[1:])
  File "scripts/manage_queues.py", line 74, in main
    queues.purge_all()
  File "/Users/daniel/azul2/src/azul/queues.py", line 353, in purge_all
    self.purge_queues_safely(self.azul_queues())
  File "/Users/daniel/azul2/src/azul/queues.py", line 356, in purge_queues_safely
    self.manage_lambdas(queues, enable=False)
  File "/Users/daniel/azul2/src/azul/queues.py", line 447, in manage_lambdas
    assert False, queue_name
AssertionError: azul-tallies_fail-daniel.fifo
```